src/SWP/Bundle/CoreBundle/Model/FacebookPage.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Superdesk Web Publisher Core Bundle.
+ *
+ * Copyright 2016 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2016 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\CoreBundle\Model;
+
+use SWP\Bundle\FacebookInstantArticlesBundle\Model\Page;
+use SWP\Component\MultiTenancy\Model\TenantAwareInterface;
+use SWP\Component\MultiTenancy\Model\TenantAwareTrait;
+use SWP\Component\Storage\Model\PersistableInterface;
+
+class FacebookPage extends Page implements TenantAwareInterface, PersistableInterface
```

Comment:
Parent Page class should implement `PersistableInterface` not this one. Plus this class missing interface which should be placed in CoreBundle, in the same level as this class. This interface should extend the one from FBIABundle. Then FacebookPage class would then implement interface from CoreBundle because atm there is a mess with it. You wont be able to re-use anything from FBIABundle later or would need to require not needed dependencies just to get one interface or class and you wont be able to create other implementation.